truffle/com.oracle.truffle.api.debug/src/com/oracle/truffle/api/debug/Debugger.java

```diff
@@ -274,8 +274,8 @@ public Breakpoint setTagBreakpoint(int ignoreCount, com.oracle.truffle.api.instr
      * </ul>
      */
     @TruffleBoundary
-    void prepareContinue(int depth) {
-        getCurrentDebugContext().setAction(depth, new Continue());
+    void prepareContinue(DebugExecutionContext context, int depth) {
```

Comment:
Why you need "DebugExecutionContext public" as your commit message says? It seems to me that all changes in this changeset are only modifying classes in `com.oracle.truffle.api.debug` package - e.g. no API changes are necessary.